Editor/AGS.Editor/Panes/ScintillaWrapper.cs

```diff
@@ -2450,12 +2449,23 @@ private void ScintillaWrapper_MouseUp(object sender, MouseEventArgs e)
                 {
                     menu.Items.Add(new ToolStripSeparator());
                 }
-                menu.Items.Add(new ToolStripMenuItem("Cut", Factory.GUIController.ImageList.Images["CutIcon"], onClick, CONTEXT_MENU_CUT));
-                menu.Items[menu.Items.Count - 1].Enabled = this.CanCutAndCopy();
-                menu.Items.Add(new ToolStripMenuItem("Copy", Factory.GUIController.ImageList.Images["CopyIcon"], onClick, CONTEXT_MENU_COPY));
-                menu.Items[menu.Items.Count - 1].Enabled = this.CanCutAndCopy();
-                menu.Items.Add(new ToolStripMenuItem("Paste", Factory.GUIController.ImageList.Images["PasteIcon"], onClick, CONTEXT_MENU_PASTE));
-                menu.Items[menu.Items.Count - 1].Enabled = this.CanPaste();
+
+                ToolStripMenuItem menuItem;
+
+                menuItem = new ToolStripMenuItem("Cut", Factory.GUIController.ImageList.Images["CutIcon"], onClick, CONTEXT_MENU_CUT);

```

Comment:
Question, what about use the `using` keyword here? Like `using( var item = new ....){...item.Enabled = ...}`, I feel it would nicely group things but I don't know if it's the correct word.

Reply:
I wasn't super familiar with the using stement in C# so [I looked up the doc](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/statements/using), and it seems that the use is when the object needs to be disposed after being used. But in this case, the ToolStripMenuItem doesn't need to be disposed until the context menu is  disposed, since it's being added to a list and rendered later.

Reply:
If the goal is to group things you can use brackets without attaching them to any keywords

```cs
var something = ....

{
    var foo = Factory.Create<IFoo>();
    foo.Bar = ?;
}
```

People usually don't do that though, keeping variable declaration and property assignments together is often considered enough